test/EasyPost/RefundTest.php

```diff
@@ -92,9 +90,9 @@ public function testRetrieve(object $refunds)
     {
         VCR::insertCassette('refunds/retrieve.yml');
 
-        $refund = Refund::retrieve($refunds['refunds'][0]);
+        $retrieved_refund = Refund::retrieve($refunds['refunds'][0]);
 
-        $this->assertInstanceOf('\EasyPost\Refund', $refund);
-        $this->assertStringMatchesFormat('rfnd_%s', $refund->id);
+        $this->assertInstanceOf('\EasyPost\Refund', $retrieved_refund);
+        $this->assertEquals($refunds['refunds'][0]->id, $retrieved_refund->id);
```

Comment:
We should able to compare objects for **testRetrieve()** in RefundTest. I was able to compare objects in Java.
```suggestion
        $this->assertEquals($refunds['refunds'][0], $retrieved_refund);
```

Reply:
I tried this one here but wasn't able to make it work due to how PHP's VCR works.